edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -27,6 +27,8 @@ public class ModuleManagementHttpClientTest : IClassFixture<EdgeletFixture>
 
         public static IEnumerable<object[]> VersionMap => GetVersionMap();
 
+        public static IEnumerable<object[]> VersionDateTimeMap => GetInvalidDateTime();
```

Comment:
Should this name imply that this map structure contains invalid datetimes?

Reply:
Fixed